tests/foreman/virtwho/test_api_virtwho.py

```diff
@@ -107,3 +111,243 @@ def test_positive_deploy_configure_id(self):
         vhd.delete()
         self.assertEqual(entities.VirtWhoConfig().search(
             query={'search': 'name={}'.format(name)}), [])
+
+    @tier1
```

Comment:
Looks more related to `tier2` .. what do you think ?

Reply:
@eko999 what is your opinion? Thanks

Reply:
Please refer to 'Test metadata documentation' to decide the tier. In general, only the case is critical, it should be Tier1.

Reply:
thanks, the CaseImportance is high,  will change to `tie2`